tekton/release.sh

```diff
@@ -0,0 +1,321 @@
+    echo;
+  }
+}
+
+createRegistrySecret() {
+  kubectl create namespace ${HUB_CI_NAMESPACE} 2>/dev/null || true
+
+  kubectl -n ${HUB_CI_NAMESPACE} delete secret registry-sec --ignore-not-found
+  kubectl -n ${HUB_CI_NAMESPACE} get secret registry-sec 2>/dev/null >/dev/null || {
+
+    echo; echo "Enter Quay registry credentials to push the images: (quay.io/tekton-hub) "
+    read -r -e -p "Enter Username: " USERNAME
+    read -r -e -sp "Enter Password: " PASSWORD
+
+    kubectl -n ${HUB_CI_NAMESPACE} create secret generic registry-sec \
+      --type="kubernetes.io/basic-auth" \
+      --from-literal=username="${USERNAME}" \
+      --from-literal=password="${PASSWORD}"
+
+    kubectl -n ${HUB_CI_NAMESPACE} annotate secret registry-sec tekton.dev/docker-0=quay.io
+  }
+}
+
+createNecessaryRoles() {
+
+  echo; echo 'Creates service account and necessary role to create resources: '
+
+  kubectl -n ${HUB_CI_NAMESPACE} delete serviceaccount registry-login --ignore-not-found
+  cat <<EOF | kubectl -n ${HUB_CI_NAMESPACE} create -f-
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: registry-login
+secrets:
+  - name: registry-sec
+EOF
+
+  kubectl -n ${HUB_NAMESPACE} delete role hub-pipeline --ignore-not-found
+  kubectl -n ${HUB_NAMESPACE} delete rolebinding hub-pipeline --ignore-not-found
+  kubectl -n ${HUB_NAMESPACE} create role hub-pipeline \
+    --resource=deployment,services,pvc,job \
+    --verb=create,get,list,delete,patch
+  kubectl -n ${HUB_NAMESPACE} create rolebinding hub-pipeline \
+    --serviceaccount=${HUB_CI_NAMESPACE}:registry-login \
+    --role=hub-pipeline
+
+  if [ "${CLUSTER}" == "openshift" ]; then
+    oc adm policy add-scc-to-user privileged system:serviceaccount:${HUB_CI_NAMESPACE}:registry-login
+
+    kubectl -n ${HUB_NAMESPACE} delete role hub-pipeline-route --ignore-not-found
+    kubectl -n ${HUB_NAMESPACE} delete rolebinding hub-pipeline-route --ignore-not-found
+    kubectl -n ${HUB_NAMESPACE} create role hub-pipeline-route \
+      --resource=route \
+      --verb=create,get,list,delete,patch
+    kubectl -n ${HUB_NAMESPACE} create rolebinding hub-pipeline-route \
+    --serviceaccount=${HUB_CI_NAMESPACE}:registry-login \
+    --role=hub-pipeline-route
+  fi
+
+  echo;
+}
+
+startPipelines() {
+  echo 'Install Tasks: '
+
+  kubectl -n ${HUB_CI_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.2/git-clone.yaml
+  kubectl -n ${HUB_CI_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/buildah/0.2/buildah.yaml
+  kubectl -n ${HUB_CI_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/golangci-lint/0.1/golangci-lint.yaml
+  kubectl -n ${HUB_CI_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/kubernetes-actions/0.2/kubernetes-actions.yaml
+  kubectl -n ${HUB_CI_NAMESPACE} apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/npm/0.1/npm.yaml
```

Comment:
I guess bundles are now enabled on dogfooding cluster so maybe we can remove these and directly reference the task via bundles. WDYT?

Reply:
Yes that would be really very nice to use

Reply:
if we do so we need to make sure that bundles are enabled on pipeline installation

Reply:
yep will add in readme